com.arviceblot.eso-addon-manager.json

```diff
@@ -0,0 +1,47 @@
+{
+    "app-id": "com.arviceblot.eso-addon-manager",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "22.08",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command": "eso-addon-manager",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--filesystem=host"
+    ],
+    "build-options": {
+        "append-path": "/usr/lib/sdk/rust-stable/bin"
+    },
+    "modules": [
+        {
+            "name": "eso-addon-manager",
+            "buildsystem": "simple",
+            "build-options": {
+                "env": {
+                    "CARGO_HOME": "/run/build/eso-addon-manager/cargo"
+                }
+            },
+            "build-commands": [
+                "cargo --offline fetch --manifest-path Cargo.toml --verbose",
+                "cargo --offline build --release --verbose",
+                "install -Dm755 ./target/release/eso-addon-manager -t /app/bin/",
+                "install -Dm644 ./data/${FLATPAK_ID}.desktop -t /app/share/applications/",
+                "install -Dm644 ./data/${FLATPAK_ID}.svg -t /app/share/icons/hicolor/scalable/apps/"
```

Comment:
I don't see the required Appstream file being installed.

Reply:
It was unclear to me from the submission requirements whether the Appstream file was required. I'll work on putting one together.

Reply:
it's listed on the requirement page....

Reply:
Yes, under the [Acceptible](https://docs.flathub.org/docs/for-app-authors/requirements#acceptable-but-should-be-submitted-upstream) section. If it is required it should be listed under [Required](https://docs.flathub.org/docs/for-app-authors/requirements#required-files). I'll work on making one anyways.